install/kubernetes/helm/istio/values-istio-all.yaml

```diff
@@ -16,6 +16,7 @@ grafana:
 
 zipkin:
   enabled: true
+  jaeger: true
```

Comment:
is the implication here that for all deployments, not just tests, we will use jaeger instead of zipkin?

it _feels_ weird to have jaeger under zipkin config, but I'm not sure how much that really matters in these templates.

Reply:
Not really - I added it here so it is picked up by tests. Not sure yet what to do about release - I was thinking this file should be dropped anyways, users should use helm template instead. 

But it's a good question if we should have zipkin or jaeger by default - jaeger is CNCF project and seems much lighter.

Re. naming: yes, plan is to rename zipkin to 'tracing' - but keep the name so upgrades/switches work. A user should be able to switch from one to the other with minimal disruption.